scripts/uptimerobot.js

```diff
@@ -0,0 +1,36 @@
+// Description:
+//    Send a message from a UptimeRobot webhook
+//    You need to add the room attribute to your hook (for example
+//    ops@muc.example.com)
+//
+//     Example UptimeRobot webhook (the ampersand is required):
+//     https://example.com/incoming/uptimerobot/secret?room=ops@example.com&
+//
+//     Example request:
+//     https://example.com/incoming/uptimerobot/secret?room=ops@example.com&monitorFriendlyName=Main%20site&alertTypeFriendlyName=Up&monitorID=23779464475
+//
+// Configuration:
+//   WEBHOOK_TOKEN: A string for building your secret webhook URL
+//
+// Authors:
+//   Greg Karékinian <greg@5apps.com>
```

Comment:
These headers can only be used for the main robot file, but I think they're also pretty much obsolete with scripts being managed as npm modules now.